### PR TITLE
make onlyoffice work again

### PR DESCRIPTION
**Pull request by szaimen**
this seems to be needed to make it work again.

### CODE REVIEW

**Comment by szaimen** [comment]
@enoch85 this is the last one. have already successfull tested it. So no more PRs are needed to make it work again.

File: old/onlyoffice.sh [hunk 1]
```diff
@@ -170,20 +174,20 @@ fi
...
-    if [ ! -f "$DHPARAMS" ]
+    if [ ! -f "$DHPARAMS_SUB" ]
     then
-        openssl dhparam -dsaparam -out "$DHPARAMS" 4096
+        openssl dhparam -dsaparam -out "$DHPARAMS_SUB" 4096
     fi
     printf "%b" "${IGreen}Certs are generated!\n${Color_Off}"
     a2ensite "$SUBDOMAIN.conf"
-    restart_webserver
+    check_command systemctl reload apache2
```
Comment (enoch85) [inline old/onlyoffice.sh:186]: Please revert this change, we use a function to only change on one place instead of several in case we need to change method.

Comment (szaimen) [reply old/onlyoffice.sh]: Is reverted 👍
